lib/network/modules/components/nodes/util/CircleImageBase.js

```diff
@@ -84,6 +89,7 @@ class CircleImageBase extends NodeBase {
       this.width = width;
       this.height = height;
       this.radius = 0.5 * this.width;
+      this.refreshNeeded = false;
```

Comment:
The purpose of this line is to turn off the indicator variable after a triggered resize calculation.
As far as I can tell, all the required resizing has been done in this block before this line. The value is set to false to prevent the resize calculation from triggering continuously.

In summary, when disregarding the rest of the logic, what happens here is:

```
if (refreshNeeded === true) {
  // Do the resize calculation
  refreshNeeded = false;
}
```

I hope this explains the intention. I believe this code is correct.